### PR TITLE
Bugfix: Ajax add header mangling

### PR DESCRIPTION
**Pull request by bollinger**
When an ajax update adds a new header css link, the client side javascript incorrectly adds attributes "id" and "type".

expected header
```
<head>
  <link href="status.css" rel="stylesheet">
</head>
```

before fix, the header is 
```
<head>
  <link type="null" href="status.css" rel="stylesheet" id="null">
</head>
```

The fix is to only add the attributes supplied by the server.

see https://github.com/bollinger/wicket-style-bug for a quick start which demonstrates the problem.

### CODE REVIEW

File: wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js [hunk 1]
```diff
@@ -2117,11 +2117,11 @@
...
-						// copy required attributes
-						css.id = node.getAttribute("id");
-						css.rel = node.getAttribute("rel");
-						css.href = node.getAttribute("href");
-						css.type = node.getAttribute("type");
+						// copy supplied attributes only.
+						for (var attrIdx = 0; attrIdx < node.attributes.length; attrIdx++) {
+							var attr = node.attributes[attrIdx];
+							css.setAttribute(attr.name, attr.value)
+						}
```
Comment (martin-g) [inline wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: Let's use jQuery for that.
See http://stackoverflow.com/a/14645827/497381 for a way to list the attributes and then use `$.attr()` to set it.

Comment (bollinger) [inline wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: What is the advantage of using jQuery over direct dom manipulation? 
IE 5.5 is dead so using element.attributes is fine?
https://developer.mozilla.org/en-US/docs/Web/API/Element/attributes

Comment (martin-g) [inline wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: There must be a reason why jQuery 3.1.0 still has this method.

**Comment by martin-g** [comment]
Thank you, @bollinger !

**Pull request closed without merging**